src/typings/user.ts

```diff
@@ -0,0 +1,23 @@
+export interface User {
```

Comment:
if those interfaces are global, the place is `src/utils/types.ts`. Also this is not the standard we use for types. if you need types, then create a file in the desired place named `types.ts`